### DISCUSSION TITLE
build for projects that do not contain a main function but in cmd/ folder instead

### DISCUSSION THREAD

**Discussion by suntong** [goreleaser/goreleaser#2512] [Q&A] [answered]
For all those projects that do not contain a main function at root but in cmd/ folder instead, they'll get

`  ⨯ release failed after 0.05s error=build for xxx does not contain a main function`

How to overcome it? Thx!

**Comment by caarlos0** [comment, marked as answer]
You can set the main property in the build config

Check the build docs in goreleaser.com 👌

**Reply by suntong** [reply]
Can you be more specific please? As I tried but found nothing -- the closest I found is https://goreleaser.com/cookbooks/using-main.version/

**Reply by caarlos0** [reply]
https://goreleaser.com/customization/build/

**Reply by suntong** [reply]
Thanks a lot sir!
